rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+ModelBase = declarative_base()
+
+
+class Game(ModelBase):
+    __tablename__ = 'game'
+
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    round_id = Column(Integer, ForeignKey('round.id'), nullable=False)
```

Comment:
I think this [previous comment](https://github.com/briana-mcn/rock-paper-scissors/pull/25#discussion_r127747698) may have gotten lost, but it still stands :)

> Since a `Game` has multiple `Round`s, this FK should be removed (only `round.game_id` is necessary to establish this relationship).